bridge/api/api.go

```diff
@@ -102,6 +102,18 @@ func (b *Api) handleMessages(c echo.Context) error {
 func (b *Api) handleStream(c echo.Context) error {
 	c.Response().Header().Set(echo.HeaderContentType, echo.MIMEApplicationJSON)
 	c.Response().WriteHeader(http.StatusOK)
+	greet := config.Message{
+		Text: "Connected",
+		Channel: "",
+		Username: "system",
+		Event: "api_connect",
```

Comment:
Could you add a `EVENT_API_CONNECTED="api_connected"` and change to 
```suggestion
		Event: config.EVENT_API_CONNECTED,
```
Any objections to only having an `event` and `timestamp` like 
`greet := config.Message{Event:config.EVENT_API_CONNECTED,Timestamp:time.Now()}`

Reply:
no objections